dev/components/header/tpl.pug

```diff
@@ -31,7 +31,8 @@ header.main-header
         .photo(
           ng-if="vm.user.photo && vm.user.photo != 'null'"
           ng-style="{backgroundImage: 'url('+ vm.user.photo + ')' }")
-        p.name.ellipsis {{vm.user.firstName + ' ' + vm.user.secondName}}
+        p.name.ellipsis(ng-if="!vm.user.firstName || !vm.user.secondName") No name
```

Comment:
Just show nothing if user doesn't have first name and last name, but if user has first or last name let's show it